### PR TITLE
Personalize throttling event support

### PR DESCRIPTION
**Pull request by shrouti1507**
Returning 429 to retry, when throttlingException is returned.

### CODE REVIEW

**Review by ruchiramoitra — COMMENTED** [review]
LGTM

**Review by ruchiramoitra — APPROVED** [review]

**Review by RanjeetMishra — COMMENTED** [review]
1. Please add description in PR.
2. Also please check build errors.

File: services/streammanager/personalize/personalizemanager.go [hunk 1]
```diff
@@ -110,8 +110,17 @@ func Produce(jsonData json.RawMessage, producer interface{}, destConfig interfac
 		statusCode := 500
 		// fetching status code from response
 		if reqErr, ok := err.(awserr.RequestFailure); ok {
-			statusCode = reqErr.StatusCode()
+			errorMessage := fmt.Sprintf(reqErr.Error())
```
[the remark this answers is not in the document]
Comment (shrouti1507) [reply services/streammanager/personalize/personalizemanager.go]: Yes, used it as `errorMessage := reqErr.Error()` as it is already returning string

**Review by ruchiramoitra — APPROVED** [review]

**Review by RanjeetMishra — APPROVED** [review]
👍

**Pull request merged**